evp.go

```diff
@@ -72,6 +80,26 @@ func cryptoHashToMD(ch crypto.Hash) (md C.GO_EVP_MD_PTR) {
 		return C.go_openssl_EVP_sha384()
 	case crypto.SHA512:
 		return C.go_openssl_EVP_sha512()
+	case crypto.SHA3_224:
+		if vMajor == 1 && (vMinor == 0 || vPatch == 0) {
```

Comment:
It might be easier to read if it is written with `>=`, as in [`SuppertsHKDF`](https://github.com/golang-fips/openssl/blob/dc825f725ab84c5598da7463c302960c0283460d/hkdf.go#L16).

Reply:
I like SupportsHKDF function, if it's ok to export like that one, I think I should use that pattern for SHA3 as well. Let me rewrite those bits.